OpenTelemetry.sln

```diff
@@ -11,20 +11,20 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		.dockerignore = .dockerignore
 		.editorconfig = .editorconfig
-		CHANGELOG.md = CHANGELOG.md
```

Comment:
Why remove CHANGELOG.md from the solution?

Reply:
There is no global CHANGELOG.md, each package has its own changelog, which will be used to generate a single/combined release notes during release.